app/assets/stylesheets/application.css.scss

```diff
@@ -21,8 +21,8 @@
 @import "bootstrap";
 /* Some Style from the Bootstrap Default Theme */
 body {
-  padding-left: 30px;
-  padding-right: 30px;
+  background-color: #263645;
```

Comment:
Color literals like `#263645` should only be used in variable declarations; they should be referred to via variable everywhere else.